conda_build/conda_interface.py

```diff
@@ -135,3 +136,160 @@ def which_prefix(path):
             # we cannot chop off any more directories, so we didn't find it
             return None
         prefix = dirname(prefix)
+
+if parse_version(conda.__version__) >= parse_version("4.3"):
+    from conda.exports import FileMode, NodeType
```

Comment:
I'm a little bit wary of adding this stuff without a test.  Can you please edit .travis.yml and add a new matrix entry for conda's master branch?